docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -42,7 +42,10 @@ Office.context.ui.displayDialogAsync('https://myAddinDomain/myDialog.html');
 
 > [!NOTE]
 > - The URL uses the HTTP**S** protocol. This is mandatory for all pages loaded in a dialog box, not just the first page loaded.
-> - The domain is the same as the domain of the host page, which can be the page in a task pane or the [function file](https://docs.microsoft.com/javascript/office/manifest/functionfile?view=office-js) of an add-in command. This is required: the page, controller method, or other resource that is passed to the `displayDialogAsync` method must be in the same domain as the host page.
+> - The dialog resource's domain is the same as the domain of the host page, which can be the page in a task pane or the [function file](https://docs.microsoft.com/javascript/office/manifest/functionfile?view=office-js) of an add-in command. This is required: the page, controller method, or other resource that is passed to the `displayDialogAsync` method must be in the same domain as the host page.
+
+> [!IMPORTANT]
+> The host page and the resources of the dialog must have the same domain. Subdomains are not considered to be part of a domain for the purposes of `displayDialogAsync`.
```

Comment:
To me, this reads the opposite of what I think it's supposed to say. The user found that when he passed `excel.myapp.com` to `displayDialogAsync`, but his add-in's domain was just `myapp.com`, it didn't work. So for purposes of displayDialogAsync, the subdomain _does matter_.  In other words, `blah.whatever.com` does NOT count as the same domain as either `nerd.whatever.com` or just `whatever.com`. The full domain, including any subdomains, has to match.